searchlib/src/vespa/searchlib/query/tree/queryreplicator.h

```diff
@@ -78,26 +78,32 @@ class QueryReplicator : private QueryVisitor {
         visitNodes(node.getChildren());
     }
 
+    void replicateMultiTerm(const MultiTerm &original, MultiTerm & replica) {
+        for (uint32_t i(0); i < original.getNumTerms(); i++) {
+            auto v = original.getAsString(i);
+            replica.addTerm(v.first, v.second);
```

Comment:
this makes the replica different from the original in some cases

Reply:
Fixed by introducing a type.